### PR TITLE
moved login screen to patternfly 5

### PR DESCRIPTION
**Pull request by edewit**

### CODE REVIEW

File: themes/src/main/resources/META-INF/keycloak-themes.json [hunk 1]
```diff
@@ -7,6 +7,6 @@
         "types": [ "login", "common", "email", "welcome" ]
     }, {
         "name" : "keycloak.v2",
-        "types": [ "account", "admin" ]
+        "types": [ "account", "admin", "login" ]
```
[the remark this answers is not in the document]
Comment (edewit) [reply themes/src/main/resources/META-INF/keycloak-themes.json:10]: No need to do that as the original doesn't change you have to pick it in the themes tab to enable it

[the remark this answers is not in the document]
Comment (jonkoops) [reply themes/src/main/resources/META-INF/keycloak-themes.json:10]: We should probably still add a feature flag to change the default it nothing is configured right? That is what we did with the other themes as well.

File: themes/src/main/resources/theme/keycloak.v2/login/theme.properties [hunk 2]
```diff
@@ -0,0 +1,22 @@
+parent=keycloak
+import=common/keycloak
+
+styles=css/login.css
```
Comment (jonkoops) [inline themes/src/main/resources/theme/keycloak.v2/login/theme.properties:4]: This loads the original login styles, which might have incompatibilities with the new stuff we're doing. Perhaps we can clear this instead?
```suggestion
styles=
```

File: common/src/main/java/org/keycloak/common/Profile.java [hunk 3]
```diff
@@ -57,6 +57,8 @@ public enum Feature {
 
         ADMIN2("New Admin Console", Type.DEFAULT, Feature.ADMIN_API),
 
+        LOGIN2("New Login Theme", Type.PREVIEW),
```
Comment (jonkoops) [inline common/src/main/java/org/keycloak/common/Profile.java:60]: I think we should probably default to `EXPERIMENTAL` here, considering this is very early code.

**Review by jonkoops — CHANGES_REQUESTED** [review]

**Review by jonkoops — DISMISSED** [review]

File: common/src/test/java/org/keycloak/common/ProfileTest.java [hunk 4]
```diff
@@ -75,6 +75,7 @@ public void checkDefaults() {
             Profile.Feature.ADMIN_FINE_GRAINED_AUTHZ,
             Profile.Feature.DYNAMIC_SCOPES,
             Profile.Feature.DOCKER,
+        Profile.Feature.LOGIN2,
```
Comment (jonkoops) [inline common/src/test/java/org/keycloak/common/ProfileTest.java:78]: Indentation here seems off.

**Review by jonkoops — DISMISSED** [review]

**Review by jonkoops — DISMISSED** [review]

**Review by jonkoops — DISMISSED** [review]

**Review by jonkoops — APPROVED** [review]

**Review by mposolda — APPROVED** [review]

**Pull request merged**